os/arch/arm/src/amebad/amebad_i2s.c

```diff
@@ -1056,7 +1113,7 @@ void i2s_transfer_rx_handleirq(void *data, char *pbuf)
 
 	int result = OK;
 
-	i2s_rxdma_callback(priv, result);
+	// i2s_rxdma_callback(priv, result);
```

Comment:
We should not disable this if we consider rx in future, how do you think?

Reply:
Hi @abhishek-samsung , could I check with you on the reason this line is commented? I see that it had been commented out during this [commit](https://github.com/abhishek-samsung/TizenRT/commit/2d164c96ddfbb916bdbf08e7a3aa71dc7278067b#diff-174f6cd168a26967c6eb71d3acb91b4e3471b6997ec3e05aa762de877c87ddb0).

Reply:
Hi @jwei5 , earlier we were not using i2s rx, so I have commented out the line. Please uncomment the line. Sorry for the confusion.

Reply:
Ok no problem, thanks for the information, I have uncommented the line.